### ISSUE TITLE
IOU is 0.000 always

### ISSUE THREAD

**Issue by sanersbug** [icoz69/CaNet#13]
In the train stage, the IOU is constant like this:

e:198 loss = 0.1104-:0.0000: 100%|██████████████| 15/15 [00:04<00:00,  3.53it/s]
----Evaluation----
IOU:0.0000
IOU for this epoch: 0.0000
best epoch:0 ,iout:0.0000
This epoch taks: 4.778814792633057 second
still need hour:0.0027
e:199 loss = 0.1190-:0.0000: 100%|██████████████| 15/15 [00:04<00:00,  3.51it/s]
----Evaluation----
IOU:0.0000
IOU for this epoch: 0.0000
best epoch:0 ,iout:0.0000
This epoch taks: 4.783169269561768 second
still need hour:0.0013

Anyone can thnow why?

**Comment by GWwangshuo** [comment]
> In the train stage, the IOU is constant like this:
> 
> e:198 loss = 0.1104-:0.0000: 100%|██████████████| 15/15 [00:04<00:00, 3.53it/s]
> ----Evaluation----
> IOU:0.0000
> IOU for this epoch: 0.0000
> best epoch:0 ,iout:0.0000
> This epoch taks: 4.778814792633057 second
> still need hour:0.0027
> e:199 loss = 0.1190-:0.0000: 100%|██████████████| 15/15 [00:04<00:00, 3.51it/s]
> ----Evaluation----
> IOU:0.0000
> IOU for this epoch: 0.0000
> best epoch:0 ,iout:0.0000
> This epoch taks: 4.783169269561768 second
> still need hour:0.0013
> 
> Anyone can thnow why?

@sanersbug Have you solved this problem?

**Comment by sanersbug** [comment]
@GWwangshuo Sorry, I give up it. How about you ?

**Comment by icoz69** [comment]
hello did you update your PyTorch to the latest version?  In my setup, all is well.

**Comment by GWwangshuo** [comment]
> hello did you update your PyTorch to the latest version? In my setup, all is well.

Thanks for your suggestion. In my case, the IoU increases after many epochs.
